### PR TITLE
Added trailer link

### PR DESCRIPTION
**Pull request by agiridharan**
#259

### CODE REVIEW

File: modules/src/movie.py [hunk 1]
```diff
@@ -23,20 +23,18 @@ def process(input, entities):
                 'include_adult': False
             })
             data = r.json()
-
             assert (len(data['results']) > 0)
             tmdb_id = str(data['results'][0]['id'])
-
             # Make another request to the API using the movie's TMDb ID to get the movie's IMDb ID
-            r = requests.get('https://api.themoviedb.org/3/movie/' + tmdb_id, params={
+            r = requests.get('https://api.themoviedb.org/3/movie/' + tmdb_id + '?api_key={api_key}&append_to_response=videos', params={
```
Comment (swapagarwal) [inline modules/src/movie.py]: These should be put in `params`.

File: modules/src/movie.py [hunk 2]
```diff
@@ -23,20 +23,18 @@ def process(input, entities):
...
                 'api_key': TMDB_API_KEY
             })
             data = r.json()
-
         template = TextTemplate('Title: ' + data['title'] +
                                 '\nYear: ' + data['release_date'][:4] +
                                 '\nAverage Rating: ' + str(data['vote_average']) + ' / 10' +
-                                '\nOverview: ' + data['overview'])
+                                '\nOverview: ' + data['overview'] + 
+                                '\nTrailer: https://www.youtube.com/watch?v=' + data['videos']['results'][0]['key'])
```
Comment (swapagarwal) [inline modules/src/movie.py]: Add this as a button.

**Comment by agiridharan** [comment]
Updated the movies.py file

File: modules/src/movie.py [hunk 3]
```diff
@@ -23,23 +23,27 @@ def process(input, entities):
...
             assert (len(data['results']) > 0)
             tmdb_id = str(data['results'][0]['id'])
-
             # Make another request to the API using the movie's TMDb ID to get the movie's IMDb ID
             r = requests.get('https://api.themoviedb.org/3/movie/' + tmdb_id, params={
                 'api_key': TMDB_API_KEY
             })
             data = r.json()
-
+            videoRequests = requests.get('https://api.themoviedb.org/3/movie/' + tmdb_id + '/videos', params={
```
Comment (swapagarwal) [inline modules/src/movie.py]: Why did we move to making a separate call? Does `append_to_response` not solve our use-case?

**Comment by swapagarwal** [comment]
Also, the merge conflict still exists. Can you rebase on current master branch?

**Comment by swapagarwal** [comment]
Closing due to inactivity. Continued in #278

**Pull request closed without merging**